src/style.css

```diff
@@ -0,0 +1,19 @@
+body {
+  margin: 0;
+}
+
+div {
+  height: 100px;
+}
+
+#red {
```

Comment:
class will be better

Reply:
when you will use BEM methodology you will always use classes, so get used to it